apps/labs/posts/intel-oneapi-msvc-spin-scipy.md

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Enhancing Developer Experience at SciPy - Intel oneAPI/MSVC Support and Migrating to spin'
+published: December 19, 2024
+authors: [gagandeep-singh]
+description: 'Highlights the work done to improve developer experience at SciPy, specifically on supporting Intel oneAPI/MSVC and spin'
+category: [Developer workflows]
+featuredImage:
+  src: /posts/intel-oneapi-msvc-spin-scipy/scipy_logo_img_featured.png
+  alt: 'SciPy logo'
+hero:
+  imageSrc: /posts/intel-oneapi-msvc-spin-scipy/scipy_logo_img_hero.png
+  imageAlt: 'SciPy logo'
+---
+
+In this blog post, we will be sharing the work we did to enhance developer experience at SciPy. We worked on two major things,
+
+1. Intel oneAPI and MSVC Support in SciPy
+2. Moving away from `dev.py` and migrating to `spin`.
+
+First, let me share the motivation behind these.
```

Comment:
It would be better I think to share the general motivation in fewer words. In-depth paragraphs about both topics before repeating part of the in-depth content in the separate sections on OneAPI and `spin` feels a little off.